File: Find and Print Max/find_max.py
# Function that finds the max value in an array and returns that value
def find_max(arr):
    print(f"find_max({arr})")
    if len(arr) == 0: # edge case
        print("list is empty")
        return
    max = arr[0]
    for i in range(len(arr)):
        if arr[i] > max:
            max = arr[i]
    print(f"find_max returned {max}")
    return max

find_max([-3,0,6,3])
find_max([6])
find_max([])

File: Find and Print Max/find_max.js
// Function that finds the max value in an array and returns that value
function find_max(arr){
    console.log("find_max(",arr,")")
    let max = arr[0]
    // Edge case
    if(arr.length==0){
        console.log("array is empty")
        return
    }
    for(i=0;i<arr.length;i++){
        if(arr[i]>max){
            max = arr[i]
        }
    }
    console.log("find_max returned", max)
    return max
}

find_max([-3,0,6,3])
find_max([6])
find_max([])